lib/fix.js

```diff
@@ -0,0 +1,18 @@
+/** @babel */
+import lint from './lint';
+
+// (editor: Object) => function
```

Comment:
Ok. Was just curious. It's fine like this. Makes the code more readable. At some point I'll probably convert the codebase to Flow type.